From: Nelson Elhage <nelhage@nelhage.com>
Date: Sun, 14 Oct 2012 22:45:45 -0700
Subject: Install more header files.

---
 Makefile |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Makefile b/Makefile
index a3de7a7..f6dab6c 100644
--- a/Makefile
+++ b/Makefile
@@ -49,6 +49,13 @@ INSTALL_HFILES=\
 	re2/set.h\
 	re2/stringpiece.h\
 	re2/variadic_function.h\
+	re2/walker-inl.h\
+	re2/util.h\
+	re2/arena.h\
+	re2/logging.h\
+	re2/mutex.h\
+	re2/utf.h\
+	re2/regexp.h
 
 HFILES=\
 	re2/arena.h\
-- 
